src/sync/remote_mongo_collection.hpp

```diff
@@ -315,17 +319,16 @@ class RemoteMongoCollection {
      */
 
 private:
-    friend class RemoteMongoDatabase;
-
-    RemoteMongoCollection(std::string name,
-                          std::string database_name,
-                          std::shared_ptr<AppServiceClient> service,
-                          std::string service_name)
-    : m_name(name)
-    , m_database_name(database_name)
-    , m_base_operation_args({ { "database" , database_name }, { "collection" , name } })
-    , m_service(service)
-    , m_service_name(service_name)
+    friend class MongoDatabase;
+
+    MongoCollection(std::string name, std::string database_name, std::shared_ptr<SyncUser> user,
+                    std::shared_ptr<AppServiceClient> service, std::string service_name)
+        : m_name(std::move(name))
```

Comment:
In general our constructor inits has been on the same indentation as the constructor declaration.